src/Path/Pretty.hs

```diff
@@ -92,15 +113,31 @@ instance Pretty Column where
   pretty = snd . unColumn
 
 
-prettyParens :: Bool -> Doc -> Doc
+prettyParens :: Bool -> PP.Doc ann -> PP.Doc ann
 prettyParens True = parens
 prettyParens False = id
 
-prettyBraces :: Bool -> Doc -> Doc
+prettyBraces :: Bool -> PP.Doc ann -> PP.Doc ann
 prettyBraces True = braces
 prettyBraces False = id
 
 
+red, yellow, green, cyan, blue, magenta :: Doc -> Doc
+red     = annotate $ color Red
+yellow  = annotate $ color Yellow
+green   = annotate $ color Green
+cyan    = annotate $ color Cyan
+blue    = annotate $ color Blue
+magenta = annotate $ color Magenta
```

Comment:
These combinators don’t exist in `prettyprinter-ansi-terminal` so we’ve defined them ourselves.